Documentation on ingesting GitLab file contents (#1496)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab/examples.md

After creating the blueprints and saving the integration configuration, you will see new entities in Port matching your projects alongside their `README.md` file contents and merge requests.

## Mapping files and file contents

> In the following example you will ingest dependencies from a package.json file in your GitLab repository to Port, you may use the following Port blueprint definitions and integration configuration:

Reviewer: ```suggestion
The following example shows how to ingest dependencies from a `package.json` file in your GitLab repository into Port.  
You can use the following Port blueprint definitions and integration configuration:
```
